pkg/detectors/image4/image4.go

```diff
@@ -0,0 +1,73 @@
+package image4
+
+import (
+	"context"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"image4"}) + `\b([0-9a-zA-Z]{22}\==)`)
```

Comment:
This looks like a base64 encoded token. The `==` at the end is padding, and might not always be like that.

This would be more correct:
`\b([0-9a-zA-Z]{22}[0-9a-zA-Z\=]{2})`

Reply:
I just commited the changes in regex